src/frontend/src/pages/SettingsPage/UserSettings/UserSettingsView.tsx

```diff
@@ -10,6 +10,25 @@ interface UserSettingsViewProps {
   settings: UserSettings;
 }
 
+const renderSlackId = (settings: UserSettings) => {
+  return settings.slackId === undefined ? (
```

Comment:
should probably do a !settings.slackId check in case it's null or empty string or something